examples/rabbitmq-transaction-producer/rabbitmq_transaction_producer.bal

```diff
@@ -14,5 +14,4 @@ public function main() returns error? {
                                                     routingKey: "MyQueue" });
         error? result = commit;
```

Comment:
Shouldn't this be returned?

Reply:
I can add `check` here I think right. Since it is inside a transaction block. Like this in this https://ballerina.io/learn/by-example/transaction-statement
I'm guessing that returns.

Reply:
Yeah, check would do.

Reply:
Fixed in 85149c18fcdeac078aaeeef4ad7ad399947a3bae